### ISSUE TITLE
Redstone Interaction

### ISSUE THREAD

**Issue by Azercoco** [AztechMC/Modern-Industrialization#342]
- [ ] Redstone Cable with MI System
- [ ] Controlable Machine with Redstone

**Comment by eliliam** [comment]
I'm on a playthrough of the StaTech Industry modpack and am hitting a point where I really would love to be able to toggle both machines as well as cables(enabling or disabling their input/output from happening). Is this something that is close?

**Comment by Swedz** [comment]
You can do that with a energy storage unit with a redstone control module

**Comment by eliliam** [comment]
Does that work for item input/output? I'm trying to disable the automatic export of resources out of a machine via a lever so that I can manually pull out the output instead. My goal is to have this lever disable the I/O of the pipe when it's enabled.

**Comment by Swedz** [comment]
No, machines still auto extract even if they are turned off by redstone

**Comment by AntonKV-TG** [comment]
It would be nice if the mod had redstone pipes, and that they could be used to control the operation of other pipes. Because without this, problems arise, for example, with turning off machines when a certain item is above a threshold.
